tests/unit/test_repos_repo.py

```diff
@@ -58,6 +58,64 @@ def test_asset_requires_a_positive_id(self):
 
         assert self.session.get.called is False
 
+    def test_create_fork(self):
+        """Verify the request to fork a repository."""
+        self.instance.create_fork()
+        self.post_called_with(
+            url_for('forks')
+        )
+
+    def test_create_hook(self):
```

Comment:
Can you also add tests to `TestRepositoryRequiresAuth` for each of these since they each require the user to authenticate?